model/entity/resource.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Nooku Framework - http://nooku.org/framework
+ *
+ * @copyright   Copyright (C) 2011 - 2015 Johan Janssens and Timble CVBA. (http://www.timble.net)
+ * @license     GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link        http://github.com/joomlatools/joomlatools-framework-activities for the canonical source repository
+ */
+
+/**
+ * Resource Model Entity.
+ *
+ * @author  Arunas Mazeika <https://github.com/amazeika>
+ * @package Koowa\Component\Activities
+ */
+class ComActivitiesModelEntityResource extends KModelEntityRow
+{
+    public function save()
```

Comment:
@amazeika This makes the row immutable right? Should we consider having a KModelEntityRowImmutable for that scenario?